Dockerfile

```diff
@@ -162,6 +162,27 @@ COPY . /work
 WORKDIR /work
 RUN ./pkg/rpm/build.sh
 
+FROM rockylinux:9 AS centos9-build
+
+RUN set -x; yum -y update && \
```

Comment:
I think this should maybe be `dnf -y update`, would you be able to test it with that?

Reply:
Changed in: [adf5c88](https://github.com/GoogleCloudPlatform/ops-agent/pull/806/commits/adf5c884475ee24fdd4be1855b47e4feaeae8bfb)